WebView/app/src/main/res/layout/activity_main.xml

```diff
@@ -15,20 +15,14 @@
   ~ limitations under the License.
   -->
 
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Do we need a FrameLayout?

Reply:
I used a Frame layout because we only have one child view. What kind of layout would you suggest?

Reply:
Why not just let WebView be the top-level view?